.github/workflows/ci.yml

```
@@ -202,6 +202,10 @@ jobs:
       run: .github/scripts/check_docs.sh
     - name: Test gifs
       run: gifs/generate_gifs.sh $(ls gifs/scenarios/)
+    - name: Build binary
+      run: ./mill copyTo cli.launcher ./scala-cli 
+    - name: Check examples
+      run: ./scala-cli --jvm temurin:17  .github/scripts/check_examples.sc
```

This ought to work I think:
```suggestion
      run: bash ./scala-cli --jvm temurin:17  .github/scripts/check_examples.sc
```